jenkins-pipelines/perf-regression-latency-ycsb-1M-rows.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    region: "us-east-1",
+    test_name: "ycsb_performance_regression_test.YCSBPerformanceRegression1MRecordsTest",
+    test_config: '''["test-cases/performance/ycsb/perf-base.yaml",]''',
+    sub_tests: ["test_latency"],
+    timeout: [time: 420, unit: "MINUTES"]
```

Comment:
for master branch, this is not necessary, but most likely for the back port it will be

Reply:
Don't know if this needs backport yet....

Reply:
BTW @fgelcer after you move the email sending to the pipeline, did you tested the `test_latency` ? that should be sending multiple emails in one job ?

Reply:
yes, of course... all the emails are sent as before, just now from the pipeline stage, and not the test itself...
one thing i have seen though is that sometimes we fail to save the `email_data.json` for some reason, so no email is sent...

Reply:
thanks @fgelcer (I've missed the part it's appending to the json, hence implemented back the send immediate mail)

Reply:
as for the timeout, I've removed it, since I don't think I'll be backporting this.